quinn/src/lib.rs

```diff
@@ -6,7 +6,7 @@
 //! builds on top of quinn-proto, which implements protocol logic independent of any particular
 //! runtime.
 //!
-//! The entry point of this crate is the [`Endpoint`](generic/struct.Endpoint.html).
+//! The entry point of this crate is the [`Endpoint`](struct.Endpoint.html).
```

Comment:
This is crate! Presumably now we could do with just `` [`Endpoint`] ``, want to try that?

Reply:
You're right! Just changed it to that.